app/models/push.rb

```diff
@@ -34,11 +34,18 @@ def validate_status
       end
       return if invalid_docs.blank?
 
-      add_status_error(invalid_docs, Asset::VALIDATION_STATUSES[:missing_children])
+      Asset::VALIDATION_STATUSES.each_pair do |status_key, status_value|
+        next if status_key == :valid
+        add_status_error(invalid_docs, status_value)
+      end
     end
 
     def add_status_error(invalid_docs, status)
-      ids_matching_status = invalid_docs.select { |doc| doc.validation_status_for_aapb.include?(status) }.map(&:id)
+      if status == Asset::VALIDATION_STATUSES[:empty]
+        ids_matching_status = invalid_docs.select { |doc| doc.validation_status_for_aapb.blank? }.map(&:id)
+      else
+       ids_matching_status = invalid_docs.select { |doc| doc.validation_status_for_aapb.include?(status) }.map(&:id)
```

Comment:
1. Single space indentation instead of double 
2. Is this variable visible further down in the method? I assume so based on the screenshots, but I thought declaring a variable in a block meant it wasn't in scope outside of the block 🤔

Reply:
It does seem strange that it is visible later but it definitely is. Regardless, to be more clear, I will initialize it outside of the if block.